libretroshare/src/pqi/p3historymgr.cc

```diff
@@ -99,11 +102,25 @@ void p3HistoryMgr::addMessage(const ChatMessage& cm)
 			enabled = true;
 		}
 
-		if(cm.chat_id.isDistantChatId())
+		if(cm.chat_id.isDistantChatId()&& mDistantEnable == true)

```

Comment:
This can be 

if(cm.chat_id.isDistantChatId() && mDistantEnable)

... since it is a boolean already.

Reply:
did I write this myself??

Reply:
No thats added by me, i saw from other example by default 
` if (cm.chat_id.isPeerId() && mPrivateEnable == true) `